### PR TITLE
Add task to download SSL certs from DNSimple

### PR DESCRIPTION
**Pull request by nullobject**
This PR adds a task to download our certs from the DNSimple API: `ssl cert downloaded`.

I also updated our nginx vhost config to use this downloaded cert, rather than the certs we bundle in our app directories. We are deprecating those certs and they will be removed shortly.

### CODE REVIEW

File: nginx.rb [hunk 1]
```diff
@@ -24,6 +24,16 @@ def vhost_link
...
   end
+
+  def cert_name
+    # Grab the TLD from the domain (e.g. dw.theconversation.com ->
+    # theconversation.com).
+    cert_domain = domain.to_s
+      .match(/([^.]+)\.([^.]+)$/).to_s
+      .gsub('.', '_')
+
+    "STAR_#{cert_domain}"
```
Comment (jameshill) [inline nginx.rb]: just curious why the `STAR_` prefix?

Comment (jameshill) [reply nginx.rb]: wildcard?

Comment (nullobject) [reply nginx.rb]: Yeah. I just kept things consistent with the DNSimple naming. They call it `STAR_theconversation_com`, etc.

Comment (jameshill) [reply nginx.rb]: ah nice one 👍

**Pull request merged**